ProcessOptimizer/space/space.py

```diff
@@ -420,14 +462,50 @@ def lhs_arange(self, n):
         # and picking the middle of each piece. Samples are spaced 1/n apart
         # inside the interval with a buffer of half a step size to the extremes
         samples = (np.arange(n)+0.5)/n
-        # Transform the samples to the range used for this dimension and then 
-        # round them back to integers. If your space is less than n wide, some 
+        # Transform the samples to the range used for this dimension and then
+        # round them back to integers. If your space is less than n wide, some
         # of your samples will be rounded to the same number
         samples = np.round(samples*(self.high - self.low) + self.low)
-        
+
         # Convert samples to a list of integers
         return samples.astype(int)
 
+    def evenly_sample(self, num_points, border_width=None):
+        """Return `n_points` evenly spaced points from a Dimension.
+
+        Parameters
+        ----------
+        * `num_points` [int]
+            The number of points to sample from `dim`.
+
+        * `border_width` [int]
+            How wide to make the borders. If None, no borders are returned.
+
+        Returns
+        -------
+        * `xi`: [np.array]:
+            The sampled points in the Dimension.
+
+        * `xi_transformed`: [np.array]:
+            The transformed values of `xi`, for feeding to a model.
+        """
+        bounds = self.bounds
+        xi = np.linspace(bounds[0], bounds[1], num_points)
+        xi_transformed = self.transform(xi)
+        if border_width is None:
+            return xi, xi_transformed
+        border_list = []
+        transformed_bounds = self.transform(self.bounds)
+        border_distance = float(
+            np.diff(transformed_bounds)/(num_points-1)*border_width)

```

Comment:
Same PEP comment as above, the closing paranthesis of the float argument should be moved down on its own line, and inside the for-loop we should add spaces around the operators + and -.